### PR TITLE
add admin delete sandbox

### PR DESCRIPTION
**Pull request by gkorland**

### CODE REVIEW

File: app/admin/page.tsx [hunk 1]
```diff
@@ -37,6 +38,36 @@ function getUsers(props: {pageIndex:number}): Promise<User[]> {
     })
 }
 
+function deleteSandbox(row: Row<any>) {
+  let user:User = row.original
+
+  if (!user.task_arn){
+    toast({
+      title: "Error",
+      description: "No task_arn found",
```
Comment (AviAvni) [inline app/admin/page.tsx:47]: not sure the user will understand what to do with this error

File: app/admin/page.tsx [hunk 2]
```diff
@@ -37,6 +38,36 @@ function getUsers(props: {pageIndex:number}): Promise<User[]> {
...
+      'Content-Type': 'application/json'
+    }
+  })
+    .then((result) => {
+      if (result.status < 300) {
+        return result.json()
+      }
+      toast({
+        title: "Error",
+        description: result.text(),
```
Comment (AviAvni) [inline app/admin/page.tsx:64]: not sure the user will understand what to do with this error

File: app/admin/page.tsx [hunk 3]
```diff
@@ -37,6 +38,36 @@ function getUsers(props: {pageIndex:number}): Promise<User[]> {
...
+
+  if (!user.task_arn){
+    toast({
+      title: "Error",
+      description: "No task_arn found",
+    })
+    return
+  }
+
+  fetch(`/api/db/${encodeURIComponent(user.task_arn)}`, {
```
Comment (AviAvni) [inline app/admin/page.tsx:52]: why not using async/await?

File: app/api/db/sandbox.tsx [hunk 4]
```diff
@@ -16,4 +22,75 @@ export class Sandbox {
...
+            return NextResponse.json({ message: `Task delete failed, can't find region: ${regionName}` }, { status: 500 })
+        }
+
+        try {
+            const tasks = await region.ecsClient.send(new ListTasksCommand({
+                cluster: "falkordb",
+                serviceName: `falkordb-${user.id}`,
+            }))
+            await deleteService(region, user.task_arn);
+            await deleteService(region, user.task_arn);
```
Comment (AviAvni) [inline app/api/db/sandbox.tsx:79]: do we need to delete the service twice?

**Review by AviAvni — CHANGES_REQUESTED** [review]

File: app/admin/page.tsx (hunk 3 above)
Comment (gkorland) [reply app/admin/page.tsx:52]: just because this is how I've done it in other places

File: app/admin/page.tsx (hunk 2 above)
Comment (gkorland) [reply app/admin/page.tsx:64]: this is the error returned from the server

**Review by AviAvni — APPROVED** [review]

**Pull request merged**